2016-08-04 20:43:25,941 [Thread-7:18232] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\git\cms\content\views.py", line 8, in mainpage
    return render(request, 'index.htm', ctx)
  File "C:\Python27\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\lib\site-packages\django\template\loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Python27\lib\site-packages\django\template\loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: index.htm
2016-08-04 20:43:51,032 [Thread-9:18252] [django.request:170] [WARNING]- Not Found: /css/font-awesome.min.css
2016-08-04 20:43:51,032 [Thread-10:17276] [django.request:170] [WARNING]- Not Found: /css/theme.css
2016-08-04 20:43:51,032 [Thread-8:8488] [django.request:170] [WARNING]- Not Found: /css/bootstrap.css
2016-08-04 20:43:51,035 [Thread-11:6136] [django.request:170] [WARNING]- Not Found: /css/prettyPhoto.css
2016-08-04 20:43:51,039 [Thread-12:12300] [django.request:170] [WARNING]- Not Found: /css/zocial.css
2016-08-04 20:43:51,102 [Thread-13:24712] [django.request:170] [WARNING]- Not Found: /css/settings.css
2016-08-04 20:43:51,253 [Thread-14:14176] [django.request:170] [WARNING]- Not Found: /js/jquery.js
2016-08-04 20:43:51,332 [Thread-15:10644] [django.request:170] [WARNING]- Not Found: /js/bootstrap.min.js
2016-08-04 20:43:51,345 [Thread-16:13320] [django.request:170] [WARNING]- Not Found: /js/jquery.themepunch.revolution.min.js
2016-08-04 20:43:51,355 [Thread-17:18472] [django.request:170] [WARNING]- Not Found: /js/jquery.themepunch.plugins.min.js
2016-08-04 20:43:51,371 [Thread-18:21848] [django.request:170] [WARNING]- Not Found: /js/jquery.touchSwipe.min.js
2016-08-04 20:43:51,394 [Thread-19:15240] [django.request:170] [WARNING]- Not Found: /js/jquery.mousewheel.min.js
2016-08-04 20:43:51,483 [Thread-20:14468] [django.request:170] [WARNING]- Not Found: /js/jquery.prettyPhoto.js
2016-08-04 20:43:51,673 [Thread-21:10752] [django.request:170] [WARNING]- Not Found: /js/scripts.js
2016-08-04 20:43:51,686 [Thread-22:10732] [django.request:170] [WARNING]- Not Found: /js/jquery.carouFredSel-6.2.1-packed.js
2016-08-04 20:43:51,710 [Thread-23:18988] [django.request:170] [WARNING]- Not Found: /img/logo.png
2016-08-04 20:43:51,756 [Thread-24:17956] [django.request:170] [WARNING]- Not Found: /img/slider/slider1.jpg
2016-08-04 20:43:51,796 [Thread-25:11484] [django.request:170] [WARNING]- Not Found: /img/slider/robot1.png
2016-08-04 20:43:51,813 [Thread-26:26608] [django.request:170] [WARNING]- Not Found: /img/slider/rocket.png
2016-08-04 20:43:51,950 [Thread-27:26256] [django.request:170] [WARNING]- Not Found: /img/slider/flames.png
2016-08-04 20:43:51,990 [Thread-28:12084] [django.request:170] [WARNING]- Not Found: /img/slider/slider2.jpg
2016-08-04 20:43:52,033 [Thread-29:14004] [django.request:170] [WARNING]- Not Found: /img/slider/iMac.png
2016-08-04 20:43:52,085 [Thread-30:19780] [django.request:170] [WARNING]- Not Found: /img/slider/slider3.jpg
2016-08-04 20:43:52,107 [Thread-31:9432] [django.request:170] [WARNING]- Not Found: /img/slider/robot2.png
2016-08-04 20:43:52,111 [Thread-32:25176] [django.request:170] [WARNING]- Not Found: /img/slider/slider4.jpg
2016-08-04 20:43:52,256 [Thread-33:13608] [django.request:170] [WARNING]- Not Found: /img/small/s1.jpg
2016-08-04 20:43:52,286 [Thread-34:22132] [django.request:170] [WARNING]- Not Found: /img/small/s2.jpg
2016-08-04 20:43:52,326 [Thread-35:20360] [django.request:170] [WARNING]- Not Found: /img/small/s3.jpg
2016-08-04 20:43:52,371 [Thread-36:17660] [django.request:170] [WARNING]- Not Found: /img/small/s4.jpg
2016-08-04 20:43:52,404 [Thread-37:19944] [django.request:170] [WARNING]- Not Found: /img/small/s5.jpg
2016-08-04 20:43:52,414 [Thread-38:3400] [django.request:170] [WARNING]- Not Found: /img/small/s6.jpg
2016-08-04 20:43:52,542 [Thread-39:10396] [django.request:170] [WARNING]- Not Found: /img/small/s7.jpg
2016-08-04 20:43:52,563 [Thread-40:13972] [django.request:170] [WARNING]- Not Found: /img/small/s8.jpg
2016-08-04 20:49:18,585 [Thread-197:8888] [django.request:170] [WARNING]- Not Found: /index.html
2016-08-04 20:53:41,177 [Thread-321:25644] [django.request:170] [WARNING]- Not Found: /contact.html
2016-08-05 14:44:58,701 [Thread-328:18308] [django.request:170] [WARNING]- Not Found: /about.html
2016-08-06 10:47:17,701 [Thread-341:12468] [django.request:170] [WARNING]- Not Found: /services.html
2016-08-06 10:48:32,339 [Thread-387:10872] [django.request:170] [WARNING]- Not Found: /services.html
2016-08-08 13:47:09,138 [Thread-13:9888] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-08 13:57:05,364 [Thread-14:3232] [django.request:170] [WARNING]- Not Found: /team.html
2016-08-08 13:57:10,460 [Thread-15:11828] [django.request:170] [WARNING]- Not Found: /index2.html
2016-08-08 14:50:15,615 [Thread-61:5360] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\git\cms\content\views.py", line 8, in mainpage
    return render(request, 'index.html', ctx)
  File "C:\Python27\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\lib\site-packages\django\template\loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 74, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 210, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 202, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 905, in render
    bit = self.render_node(node, context)
  File "C:\Python27\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 113, in render
    compiled_parent = self.get_parent(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 103, in get_parent
    raise TemplateSyntaxError(error_msg)
TemplateSyntaxError: Invalid template name in 'extends' tag: ''. Got this from the 'base.html' variable.
2016-08-08 14:55:21,029 [Thread-140:18744] [django.request:170] [WARNING]- Not Found: /mainpage
2016-08-08 16:16:23,493 [Thread-47:18648] [django.request:256] [ERROR]- Internal Server Error: /tcs/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\git\cms\content\views.py", line 13, in tcs
    return render(request, 'tcs.html', ctx)
  File "C:\Python27\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\lib\site-packages\django\template\loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Python27\lib\site-packages\django\template\loader.py", line 46, in get_template
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: tcs.html
2016-08-08 18:31:54,519 [Thread-1739:15952] [django.request:170] [WARNING]- Not Found: /video_gallery.html
2016-08-08 18:39:19,457 [Thread-46:17032] [django.request:170] [WARNING]- Not Found: /mobsolu
2016-08-08 21:09:25,141 [Thread-396:15672] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-09 15:19:20,581 [Thread-48:3700] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-09 15:20:30,737 [Thread-90:14408] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-09 17:58:03,543 [Thread-147:16596] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-10 11:31:15,032 [Thread-48:17804] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-10 11:31:23,006 [Thread-63:21772] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-10 11:33:21,927 [Thread-135:16776] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-10 14:16:42,749 [Thread-240:25352] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-13 12:55:23,471 [Thread-48:26940] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-22 10:01:49,148 [Thread-62:11668] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-22 10:18:01,522 [Thread-83:2468] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-22 10:49:47,056 [Thread-7:1656] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-22 15:33:18,082 [Thread-92:10660] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-23 17:35:35,563 [Thread-48:1520] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-24 08:52:29,108 [Thread-7:5932] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-24 08:52:34,427 [Thread-39:3884] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-24 11:57:13,318 [Thread-7:2116] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-24 11:57:25,582 [Thread-12:1964] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-24 14:52:41,197 [Thread-68:6940] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-25 09:22:19,568 [Thread-7:6556] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-25 09:22:25,994 [Thread-43:7852] [django.request:182] [WARNING]- Not Found: /favicon.ico
2016-08-25 09:53:44,450 [Thread-51:7276] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-25 13:43:49,096 [Thread-49:17960] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-25 13:54:13,062 [Thread-210:6756] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-25 14:01:28,421 [Thread-62:24052] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-25 14:13:00,141 [Thread-278:12484] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-25 15:23:45,226 [Thread-7:10420] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:/git/portal_web/trunk\content\views.py", line 42, in mainpage
    return render(request, 'index.html', ctx)
NameError: global name 'ctx' is not defined
2016-08-25 15:24:07,000 [Thread-7:4584] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:/git/portal_web/trunk\content\views.py", line 41, in mainpage
    ctx = commonfunc('homeactive')
  File "C:/git/portal_web/trunk\content\views.py", line 8, in commonfunc
    ctx['homeactive'] = 'active'
TypeError: list indices must be integers, not str
2016-08-25 15:24:08,835 [Thread-8:24968] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:/git/portal_web/trunk\content\views.py", line 41, in mainpage
    ctx = commonfunc('homeactive')
  File "C:/git/portal_web/trunk\content\views.py", line 8, in commonfunc
    ctx['homeactive'] = 'active'
TypeError: list indices must be integers, not str
2016-08-25 15:35:30,585 [Thread-112:20308] [django.request:170] [WARNING]- Not Found: /media/banner/banner01.jpg
2016-08-25 15:35:38,690 [Thread-25:12872] [django.request:170] [WARNING]- Not Found: /media/banner/banner02.jpg
2016-08-25 16:23:11,898 [Thread-27:23148] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:23:12,365 [Thread-44:9988] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:23:13,006 [Thread-45:7152] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:23:26,914 [Thread-46:744] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:23:31,377 [Thread-47:21916] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:23:41,252 [Thread-48:12876] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:23:50,515 [Thread-49:18248] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:23:59,736 [Thread-50:24464] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:24:08,933 [Thread-51:7816] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:24:18,539 [Thread-52:22716] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:24:27,585 [Thread-53:8640] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:24:36,838 [Thread-54:10864] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:24:46,122 [Thread-55:13400] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:24:55,278 [Thread-56:13144] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:25:04,683 [Thread-57:17036] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:25:13,878 [Thread-58:16396] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:25:21,490 [Thread-59:24412] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:25:44,990 [Thread-64:17760] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:26:37,332 [Thread-8:12220] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:26:58,039 [Thread-7:21468] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:27:19,130 [Thread-7:5928] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 16:27:34,938 [Thread-7:12716] [django.request:170] [WARNING]- Not Found: /media/banner/banner03.jpg
2016-08-25 17:43:44,888 [Thread-7:11672] [django.request:256] [ERROR]- Internal Server Error: /admin/para/para/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 618, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 233, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1518, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 34, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 30, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1472, in changeform_view
    self.log_addition(request, new_object)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 812, in log_addition
    object_repr=force_text(object),
  File "C:\Python27\lib\site-packages\django\utils\encoding.py", line 92, in force_text
    s = six.text_type(s)
TypeError: coercing to Unicode: need string or buffer, NoneType found
2016-08-25 17:45:26,719 [Thread-9:19620] [django.request:256] [ERROR]- Internal Server Error: /admin/para/para/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 618, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 233, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1518, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 34, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 30, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1472, in changeform_view
    self.log_addition(request, new_object)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 812, in log_addition
    object_repr=force_text(object),
  File "C:\Python27\lib\site-packages\django\utils\encoding.py", line 92, in force_text
    s = six.text_type(s)
TypeError: coercing to Unicode: need string or buffer, NoneType found
2016-08-25 17:45:39,691 [Thread-11:20304] [django.request:256] [ERROR]- Internal Server Error: /admin/para/para/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 618, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 233, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1518, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 34, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 30, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1472, in changeform_view
    self.log_addition(request, new_object)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 812, in log_addition
    object_repr=force_text(object),
  File "C:\Python27\lib\site-packages\django\utils\encoding.py", line 92, in force_text
    s = six.text_type(s)
TypeError: coercing to Unicode: need string or buffer, NoneType found
2016-08-25 17:47:59,549 [Thread-13:23428] [django.request:256] [ERROR]- Internal Server Error: /admin/para/para/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 618, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 233, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1518, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 34, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 30, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1472, in changeform_view
    self.log_addition(request, new_object)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 812, in log_addition
    object_repr=force_text(object),
  File "C:\Python27\lib\site-packages\django\utils\encoding.py", line 92, in force_text
    s = six.text_type(s)
TypeError: coercing to Unicode: need string or buffer, NoneType found
2016-08-25 17:53:56,976 [Thread-7:23768] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:/git/portal_web/trunk\content\views.py", line 77, in mainpage
    ctx = recentword(ctx)
  File "C:/git/portal_web/trunk\content\views.py", line 70, in recentword
    ctx['recentwork'] = {'projs', projs}
TypeError: unhashable type: 'list'
2016-08-25 17:54:59,046 [Thread-29:14536] [django.request:170] [WARNING]- Not Found: /files/banner01_FZvZKZp.jpg
2016-08-25 17:55:20,066 [Thread-45:22520] [django.request:170] [WARNING]- Not Found: /files/banner01_FZvZKZp.jpg
2016-08-25 17:56:42,076 [Thread-7:16760] [django.request:170] [WARNING]- Not Found: /files/banner01_FZvZKZp.jpg
2016-08-25 17:56:49,903 [Thread-30:11800] [django.request:170] [WARNING]- Not Found: /files/banner01_FZvZKZp.jpg
2016-08-25 17:56:55,210 [Thread-68:18704] [django.request:170] [WARNING]- Not Found: /files/banner01_FZvZKZp.jpg
2016-08-25 17:57:00,158 [Thread-104:17404] [django.request:170] [WARNING]- Not Found: /files/banner01_FZvZKZp.jpg
2016-08-25 17:57:14,621 [Thread-120:25904] [django.request:170] [WARNING]- Not Found: /files/banner01_FZvZKZp.jpg
2016-08-26 16:10:14,818 [Thread-469:4544] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-08-26 17:01:25,220 [Thread-1094:9292] [django.request:170] [WARNING]- Not Found: /favicon.ico
